docs/release-notes.md

```diff
@@ -2,6 +2,23 @@
 
 This page summarizes changes for each major update in Proton and Timeplus Cloud, including new features and important bug fixes. For previous release notes, please check [this page](changelog).
 
+## Apr 1, 2024
+
+*Timeplus Proton:*
+  * Added support for selecting message key from Apache Kafka external streams, via `SELECT _message_key, * FROM external_stream`.
+  * Streaming processing now supports nullable data type.
+  * [External Table](proton-clickhouse-external-table#create-external-table): When naming ClickHouse external tables, special characters such as dashes are now supported. Simply set `table='test-a-b'` in the `CREATE EXTERNAL STREAM` DDL.
+  * [External Stream](proton-kafka#create-external-stream): Error handling and connection pooling/retry for Kafka external streams have been greatly improved.
+  * Materialized View: Added option to [skip dirty/unexpected data](query-syntax#settings). If you set `SETTINGS recovery_policy=best_effort`, Timeplus will try up to 3 times, then skip dirty data and continue processing the rest of the data.
+
+*Timeplus Cloud:*
```

Comment:
From now on, let's call this Timeplus Cloud and Timeplus Enterprise

Reply:
Thanks for your comments, @jovezhong. Both items now updated.